lore/io/connection.py

```diff
@@ -272,7 +276,21 @@ def select(self, sql=None, extract=None, filename=None, **kwargs):
 
     @query_cached
     def _select(self, sql, bindings):
-        return self.__execute(sql, bindings).fetchall()
+        if self._use_psycopg2:
+            try:
```

Comment:
Wonder if there is value in falling back to SQL alchemy as a final try before giving up completely.

Reply:
Not a blocker though. This looks good!